qcfractal/interface/collections/dataset_view.py

```diff
@@ -185,10 +231,67 @@ def _write_dataset(dataset, column, entry_dset):
                 f.attrs[field] = self._serialize_field(getattr(ds.data, field))
             f.attrs["server_information"] = self._serialize_field(ds.client.server_information())
 
+            # Export molecules
+            # TODO: duplicate molecules
+            molecule_group = f.create_group("molecule")
+
+            if "stoichiometry" in ds.data.history_keys:
+                #stoichs = list(ds.data.records[0].stoichiometry.keys())
+                molecules = ds.get_molecules(stoich=list(ds.valid_stoich), force=True)
+            else:
+                molecules = ds.get_molecules(force=True)
+            mol_shape = (len(molecules), )
+            mol_geometry = molecule_group.create_dataset("geometry",
+                                                         shape=mol_shape,
+                                                         dtype=vlen_double_t,
+                                                         **dataset_kwargs)
+            mol_symbols = molecule_group.create_dataset("symbols",
+                                                        shape=mol_shape,
+                                                        dtype=vlen_utf8_t,
+                                                        **dataset_kwargs)
+            mol_schema = molecule_group.create_dataset("schema", shape=mol_shape, dtype=utf8_t, **dataset_kwargs)
```

Comment:
Do we want to worry about additional fields like charge/multiplicity or leave that for another day?

Reply:
No time like the present!